src/lib/config/homestead.json

```diff
@@ -15,6 +15,9 @@
   "subgraph": "https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-v2",
   "poolsUrlV1": "https://storageapi.fleek.co/balancer-bucket/balancer-exchange/pools",
   "poolsUrlV2": "https://storageapi.fleek.co/johngrantuk-team-bucket/poolsV2.json",
+  "subgraphs": {
+    "aave": "https://api.thegraph.com/subgraphs/name/aave/protocol-v2"
+  },
```

Comment:
I think this approach should give us the flexibility to add multiple subgraphs in the future (and perhaps also move balancer inside at some point)